src/main/java/com/xebialabs/overthere/gcp/credentials/GcpCredentialFactory.java

```diff
@@ -18,7 +18,7 @@
      * @return credentials with additional scopes.
      */
     public ProjectCredentials withScope(ProjectCredentials projectCredentials) {
-        Credentials credentials = projectCredentials.getCredentials();
+        Object credentials = projectCredentials.getCredentials();
```

Comment:
Why is there a change in this line?

Reply:
Yes was initially using Other Type of object, forgot to change it back , will change it :)